README.md

```diff
@@ -230,6 +230,10 @@ ssl {
   // useful for self-signed certificates or for organizations using their own
   // internal certificate authority.
   ca_cert = "/path/to/ca"
+
+  // This is the path to a directory of PEM-encoded CA cert files. If both
+  // `ca_cert` and `ca_path` is specified, `ca_cert` is preferred.
+  ca_path = "path/to/certs/"
```

Comment:
Since this is a directory, I would prefer (either):

- `cas_path` (but I don't like this because I think "check-and-set"
- `ca_dir`

"path" makes me thing "path to a file", whereas "cert" makes me think "path to a place"

However - could we just allow ca_cert to be a filepath or directory path? Stat it and, if it's a directory, treat it as such?